pkg/kv/kvserver/store_split.go

```diff
@@ -105,9 +85,24 @@ func splitPreApply(
 				log.Fatalf(ctx, "failed to load hard state for removed rhs: %v", err)
 			}
 		}
-		const rangeIDLocalOnly = false
-		const mustUseClearRange = false
-		if err := clearRangeData(&split.RightDesc, readWriter, readWriter, rangeIDLocalOnly, mustUseClearRange); err != nil {
+		if err := clearRangeData(split.RightDesc.RangeID, readWriter, readWriter, clearRangeDataOptions{
+			// We know there isn't anything in these two replicated spans below in the
+			// right-hand side (before the current batch), so setting these options
+			// will in effect only mask the writes to the RHS replicated state we have
+			// staged in this batch, which is what we're after.
```

Comment:
What does "mask" mean here? Will it remove those staged writes?

Reply:
"clear",  fixed.